app/Http/Controllers/GroupController.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Judite\Models\Group;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Http\Request;
+use App\Judite\Models\Course;
+use App\Judite\Models\User;
+use Maatwebsite\Excel\Facades\Excel;
+use Illuminate\Support\Facades\Mail;
+use App\Judite\Models\Student;
+
+class GroupController extends Controller
+{
+    /**
+     * Create a new controller instance.
+     */
+    public function __construct()
+    {
+        $this->middleware('auth');
+        #$this->middleware('can.student');
+        #$this->middleware('can.group')->only(['create', 'store']);
```

Comment:
Uncomment this and add to the right methods... I'm thinking that all methods are affected by this middleware except maybe the index method